src/components/AuthForm.js

```diff
@@ -20,10 +20,24 @@ function AuthForm (props) {
       }
     }
 
-    const urlForUsers = endpointApiUrl.concat("/","users")
+    const urlForUsers = endpointApiUrl.concat("/users")
   
     let register = await axios.post(urlForUsers, data)
-    let tokenInfo = sessionStorage.setItem('jwt', register.data.user.token);
```

Comment:
`tokenInfo` variable has been removed as no values need to be stored in a variable from `sessionStorage.setItem`.